plugins/no_follow.rb

```diff
@@ -0,0 +1,32 @@
+# Jekyll Auto Nofollow Plugin
+# Automatically adds rel='external nofollow' to outgoing links.
+
+require 'jekyll'
+require 'nokogiri'
+
+module Jekyll
+    module NoFollow
+        def nofollow(content)
+            dom = Nokogiri::HTML.fragment(content)
+
+            # Find all links
+            dom.css('a').each do |link|
+
+                # All external links start with 'http', skip when this one does not
+                next unless link.get_attribute('href') =~ /\Ahttp/i
```

Comment:
This assumption is not correct. For the release notes I use absolute urls because I copy paste same notes between GitHub and our site.

Reply:
Ok, will try to work around that one...

Reply:
nevermind, just saw the home assistant check a few lines below

Reply:
Still not an issue right?

This rule is about skipping relative links...